### PR TITLE
[search] Added README for search quality.

### PR DESCRIPTION
**Pull request by ygorshenin**

### CODE REVIEW

**Comment by ygorshenin** [comment]
@mpimenov PTAL

File: search/search_quality/README.txt [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+1. Prerequisites.
+
+   * get the latest version of omim (https://github.com/mapsme/omim) and build it
+
+   * get the latest version of statistics repository (https://github.com/mapsme/statistics)
+
+   * install Common Lisp. Note that there are many implementations,
+     but we recommend to use SBCL (http://www.sbcl.org/)
+
+   * install Python 3.x, and packages for data analysis (sklearn, scipy, numpy, pandas, matplotlib)
```
Comment (mpimenov) [inline search/search_quality/README.txt]: запятая не нужна

File: search/search_quality/README.txt [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+   * download maps necessary for search quality tests.
+     For example:
+
+     ./download-maps.sh -v 160524
+
+     will download all necessary maps of version 160524 to the current directory.
+
+
+2. This section describes how to run search engine on a subset of
+real-world queries and to get CSV file with search engine output.
```
Comment (mpimenov) [inline search/search_quality/README.txt]: indent please

File: search/search_quality/README.txt [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+    i) Run gen-samples.lisp script to get search queries with list of
+       vital or relevant responses in JSON format.  For example:
```
Comment (mpimenov) [inline search/search_quality/README.txt]: лишний пробел перед For

File: search/search_quality/README.txt [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+     ./download-maps.sh -v 160524
+
+     will download all necessary maps of version 160524 to the current directory.
+
+
+2. This section describes how to run search engine on a subset of
+real-world queries and to get CSV file with search engine output.
+
+    i) Run gen-samples.lisp script to get search queries with list of
```
Comment (mpimenov) [inline search/search_quality/README.txt]: s/list/lists/ ?

File: search/search_quality/README.txt (hunk 1 above)
Comment (ygorshenin) [inline search/search_quality/README.txt]: Done

File: search/search_quality/README.txt (hunk 2 above)
Comment (ygorshenin) [inline search/search_quality/README.txt]: Done

File: search/search_quality/README.txt (hunk 4 above)
Comment (ygorshenin) [inline search/search_quality/README.txt]: Done

File: search/search_quality/README.txt (hunk 3 above)
Comment (ygorshenin) [inline search/search_quality/README.txt]: sponsored by fmt

File: search/search_quality/README.txt [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+         --stats_path /tmp/stats.txt \
+         2>/dev/null >samples.csv
+
+       runs search engine on all queries from samples.json, prints
+       useful info to /tmp/stats.txt and generates a CSV file with
+       search engine output on each query.
+
+  The result CSV file is ready for analysis, i.e. for search quality
+  evaluation, ranking models learning etc. For details, take a look at
+  scoring_model.py script.
```
Comment (mpimenov) [inline search/search_quality/README.txt]: iii) To get a quick look at what the search returns without launching the application, consider using search_quality_tool:
  search_quality_tool --viewport=moscow --queries_path=$OMIM/search/search_quality/search_quality_tool/queries.txt --top 1 2>/dev/null
By default, mwm files in $OMIM/data are used.

Comment (ygorshenin) [inline search/search_quality/README.txt]: Done

**Comment by ygorshenin** [comment]
PTAL

File: search/search_quality/README.txt [hunk 6]
```diff
@@ -0,0 +1,56 @@
+This document describes how to use all these tools for search quality analysis.
```
Comment (mpimenov) [inline search/search_quality/README.txt]: s/all these/the/ ?

Comment (mpimenov) [inline search/search_quality/README.txt]: s/tools for sqa/sqa tools/?

Comment (mpimenov) [inline search/search_quality/README.txt]: хотя нет, так хуэе

File: search/search_quality/README.txt [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+   * download maps necessary for search quality tests.
+     For example:
+
+     ./download-maps.sh -v 160524
+
+     will download all necessary maps of version 160524 to the current directory.
+
+
+2. This section describes how to run search engine on a set of search
+   queries and how to get CSV file with search engine output.
```
Comment (mpimenov) [inline search/search_quality/README.txt]: s/CSV/a CSV/

File: search/search_quality/README.txt [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+       features_collector_tool --mwm_path path-to-downloaded-maps \
+         --json_in samples.json \
+         --stats_path /tmp/stats.txt \
+         2>/dev/null >samples.csv
+
+       runs search engine on all queries from samples.json, prints
+       useful info to /tmp/stats.txt and generates a CSV file with
+       search engine output on each query.
+
+       The result CSV file is ready for analysis, i.e. for search
```
Comment (mpimenov) [inline search/search_quality/README.txt]: s/result/resulting/

File: search/search_quality/README.txt [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+       quality evaluation, ranking models learning etc. For details,
+       take a look at scoring_model.py script.
+
+  iii) To get a quick look at what the search returns without
```
Comment (mpimenov) [inline search/search_quality/README.txt]: s/get/take/

File: search/search_quality/README.txt [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+       launching the application, consider using search_quality_tool:
+
+       search_quality_tool --viewport=moscow \
+         --queries_path=path-to-omim/search/search_quality/search_quality_tool/queries.txt
```
Comment (mpimenov) [inline search/search_quality/README.txt]: возможно, лучше тут написать /tmp/queries.txt, чтобы не засорять?

File: search/search_quality/README.txt (hunk 6 above)
Comment (ygorshenin) [inline search/search_quality/README.txt]: Done

File: search/search_quality/README.txt (hunk 7 above)
Comment (ygorshenin) [inline search/search_quality/README.txt]: Done

File: search/search_quality/README.txt (hunk 8 above)
Comment (ygorshenin) [inline search/search_quality/README.txt]: Done

File: search/search_quality/README.txt (hunk 9 above)
Comment (ygorshenin) [inline search/search_quality/README.txt]: Done

File: search/search_quality/README.txt (hunk 10 above)
Comment (ygorshenin) [inline search/search_quality/README.txt]: Better to leave this as is or replace to queries.txt

Comment (ygorshenin) [inline search/search_quality/README.txt]: But we need to move queries.txt to search_quality dir for this.

**Comment by ygorshenin** [comment]
PTAL

File: search/search_quality/README.txt [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+     ./download-maps.sh -v 160524
+
+     will download all necessary maps of version 160524 to the current directory.
+
+
+2. This section describes how to run search engine on a set of search
+   queries and how to get a CSV file with search engine output.
+
+    i) Run gen-samples.lisp script to get search queries with lists of
+       vital or relevant responses in JSON format.  For example:
```
Comment (mpimenov) [inline search/search_quality/README.txt]: Лишний пробел перед "For"

**Comment by mpimenov** [comment]
LGTM

**Pull request merged**